contracts/adapters/paraswap/BaseParaSwapAdapter.sol

```diff
@@ -3,12 +3,12 @@ pragma solidity ^0.8.10;
 
 import {DataTypes} from '@aave/core-v3/contracts/protocol/libraries/types/DataTypes.sol';
 import {FlashLoanSimpleReceiverBase} from '@aave/core-v3/contracts/flashloan/base/FlashLoanSimpleReceiverBase.sol';
-import {GPv2SafeERC20} from '@aave/core-v3/contracts/dependencies/gnosis/contracts/GPv2SafeERC20.sol';
```

Comment:
Let's keep using GPv2SafeERC20 for transfers, cause its more gas effective. So add SafeERC20 and use it only for safeApprove calls

Reply:
Fixed